pvlib/ivtools/sdm.py

```diff
@@ -48,7 +48,7 @@ def fit_cec_sam(celltype, v_mp, i_mp, v_oc, i_sc, alpha_sc, beta_voc,
     beta_voc : float
         Temperature coefficient of open circuit voltage [V/C]
     gamma_pmp : float
-        Temperature coefficient of power at maximum point point [%/C]
+        Temperature coefficient of power at maximum point [%/C]
```

Comment:
```suggestion
        Temperature coefficient of power at maximum power point [%/C]
```

I think this one was an incorrect word, not an extra word